src/functionalTest/java/uk/gov/hmcts/reform/waworkflowapi/controllers/SendMessageTest.java

```diff
@@ -197,6 +205,69 @@ public void should_not_be_able_to_post_as_message_does_not_exist() {
             .statusCode(HttpStatus.INTERNAL_SERVER_ERROR_500);
     }
 
+    @Test
+    public void transition_creates_a_task_and_goes_through_external_task() {
+        String dueDate = ZonedDateTime.now().plusDays(2).format(DateTimeFormatter.ISO_LOCAL_DATE_TIME);
+        String idempotentKey =  UUID.randomUUID().toString();
+        Map<String, DmnValue<?>> processVariables = mockProcessVariables(
+            dueDate,
+            "Provide Respondent Evidence",
+            "provideRespondentEvidence",
+            "external"
+        );
+
+        processVariables.put("idempotentKey", DmnValue.dmnStringValue(idempotentKey));
+
+        given()
+            .relaxedHTTPSValidation()
+            .header(SERVICE_AUTHORIZATION, serviceAuthorizationToken)
+            .contentType(APPLICATION_JSON_VALUE)
+            .body(new SendMessageRequest(
+                "createTaskMessage",
+                processVariables,
+                null
+            )).log().body()
+            .baseUri(testUrl)
+            .basePath("/workflow/message")
+            .when()
+            .post()
+            .then()
+            .statusCode(HttpStatus.NO_CONTENT_204);
+
+        waitSeconds(2);
```

Comment:
This is quite fragile shall we use the polling mechanism instead:
https://github.com/hmcts/wa-task-management-api/blob/3635c275081b94428cc837df62c8e6bf2a5b2e0b/src/testUtils/java/uk/gov/hmcts/reform/wataskmanagementapi/config/GivensBuilder.java#L115

Reply:
I have the suggested improvement